tools/doc/common.js

```diff
@@ -34,6 +34,10 @@ function extractAndParseYAML(text) {
     meta.deprecated = arrify(meta.deprecated);
   }
 
+  if (meta.removed) {
+    meta.removed = arrify(meta.removed);
```

Comment:
Is this needed for this PR? From what I can tell we only used single `removed` fields here?

Reply:
I was blindly copy pasting wherever `added` occurred.

Maybe not for this particular PR, but if we're planning to add `removed` as a feature for api docs, would rather it supports semver-minors as well, like `added` and `deprecated`.

Reply:
@SirR4T Ah, sorry, I thought we already had `removed` before, looks like it is new in this PR...